specs/clipboard-api/clipboard-api.md

```diff
@@ -0,0 +1,312 @@
+# Decoupled API for Windows clipboard in Project Reunion SDK
+
+## Background
+
+The Windows clipboard is the component that's responsible for implementing
+the "cut", "copy", and "paste" system for moving and copying data from one
+Windows application to another. Windows maintains one clipboard per
+interactively signed-in user (more precisely, one clipboard per Remote Desktop/
+Terminal Services session), which can hold any number of data "formats"
+which are all intended to represent the same data value.
+
+This specification defines the API for the Windows clipboard within the
+Project Reunion SDK. This Windows Runtime API is a [converged API interface](
+  ../../docs/README.md#converged-apis) in the Project Reunion SDK,
+which is intended to replace the following 3 clipboard APIs within Windows:
+
+* [The classic clipboard API](https://docs.microsoft.com/windows/win32/dataxchg/clipboard),
+  including GetClipboardData() and friends in windows.h.
+
+* [The OLE data transfer API](https://docs.microsoft.com/windows/win32/com/data-transfer),
+  including OleGetClipboard() and friends in ole2.h.
+
+* [Windows.ApplicationModel.DataTransfer.Clipboard](https://docs.microsoft.com/windows/uwp/app-to-app/copy-and-paste)
+  and friends.
+
+Although this API will be based closely on Windows.ApplicationModel.DataTransfer,
+its implementation will be delivered as an open-source part of the
+Project Reunion SDK decoupled from Windows. It will _not_ be added to Windows
+as part of the [Reunion Subset API Family](../../docs/README.md#subset-api-family).
+
+### Motivation: reasons to create a new clipboard API
+
+The first version of this spec creates a new clipboard API but does not
+make any changes that add substantial value. However, creating a new
+clipboard API in the Reunion SDK will let us make future improvements
+to the clipboard developer and user experiences, like these:
+
+* **Improve clipboard reliability by removing traps in the clipboard APIs:**

```

Comment:
should also address the foreground/background access restrictions here, what is the behavior these new apis going to support